docker/README.md

```diff
@@ -3,12 +3,12 @@
 This offers a simplified build environment for testing packaging
 
 To build the image:
-`docker build -t jenkins-packaging-builder:0.2 .`
+`docker build -t jenkins-packaging-builder:0.3 .`
 
 ## To use it: ##
```

Comment:
In order to test building and publishing image, we can use a docker-compose that I recently introduced, no need for build new docker image, everything works out of the box as long as we have a jenkins.war at the root of the project as explained in this [pull request](https://github.com/jenkinsci/packaging/pull/136), basically I uses docker-compose and jenkinsciinfra/packaging docker image